src/utils/env.ts

```diff
@@ -96,7 +96,10 @@ export const ENV = {
     return castBooleanEnv('AUTH_WEBAUTHN_ENABLED', false);
   },
   get AUTH_WEBAUTHN_RP_ID() {
-    return castStringEnv('AUTH_WEBAUTHN_RP_ID', '');
+    return castStringEnv(
+      'AUTH_WEBAUTHN_RP_ID',
+      ENV.AUTH_SERVER_URL && new URL(ENV.AUTH_SERVER_URL).hostname
+    );
```

Comment:
Are there any use cases where uses don't want to deduct the `AUTH_WEBAUTHN_RP_ID` from `AUTH_SERVER_URL`? 

If no, maybe we should remove `AUTH_WEBAUTHN_RP_ID`?